src/components/VFile/VFile.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <label
+    :for="id || label"
```

Comment:
for/id is required for label/input. I created an `id` prop so people can pass in a unique ID in case they have two `<vfile>`'s on the same page with the same label.

Reply:
Ah yeah. This is good. Solved on VInput, but I guess I didnt think about this one.

Reply:
yes, ID's can have spaces in them it's just not commonly done